class.jetpack.php

```diff
@@ -3325,6 +3325,9 @@ public static function unlink_user( $user_id = null ) {
 	public static function try_registration() {
 		// The user has agreed to the TOS at some point by now.
 		Jetpack_Options::update_option( 'tos_agreed', true );
+		// Make sure that we initialize tracking if we haven't already
+		$tracking = new \Automattic\Jetpack\Plugin\Tracking();
```

Comment:
Could we remove the leading slash?

```suggestion
		$tracking = new Automattic\Jetpack\Plugin\Tracking();
```